temba/channels/views.py

```diff
@@ -1844,6 +1846,59 @@ def form_valid(self, form):
 
             return HttpResponseRedirect(reverse('channels.channel_configuration', args=[channel.id]))
 
+    class ClaimLine(OrgPermsMixin, SmartFormView):
+        class LineForm(forms.Form):
+            channel_id = forms.CharField(label=_("Channel ID"), required=True, help_text=_("The ID of the your Channel on Business LINE"))
+            channel_secret = forms.CharField(label=_("Channel Secret"), required=True, help_text=_("The Secret of the your Channel on Business LINE"))
+            channel_mid = forms.CharField(label=_("Channel MID"), required=True, help_text=_("The MID of the your Channel on Business LINE"))
+
+            def clean(self):
+                from linebot.client import LineBotClient
+
+                channel_id = self.cleaned_data.get('channel_id')
+                channel_secret = self.cleaned_data.get('channel_secret')
+                channel_mid = self.cleaned_data.get('channel_mid')
+
+                existing = Channel.objects.filter(channel_type=Channel.TYPE_LINE, address=channel_mid, is_active=True).first()
+                if existing:
+                    raise ValidationError(_("Channel with this address already exists, please try again with other."))
+
```

Comment:
Language here isn't quite right.

A Channel with this address already exists, please try again with a different address.